src/mono/mono/metadata/icall.c

```diff
@@ -985,7 +985,7 @@ ves_icall_System_Runtime_CompilerServices_RuntimeHelpers_GetSpanDataFrom (MonoCl
 		return NULL;
 	}
 
-	MonoType *type = targetTypeHandle;
+	MonoType *type = mono_type_get_underlying_type (targetTypeHandle);
```

Comment:
I assume get_underlying_type just returns the target type if it's not an enum?

Reply:
yes